### PR TITLE
Add integration tests to describe command

### PR DESCRIPTION
**Pull request by kipz**
NOTE use of timestamped rug/resolver

### CODE REVIEW

**Comment by kipz** [comment]
hmm...seems like a lot of formatting changed this time. Do you know what I should fix this time?

**Comment by cdupuis** [comment]
Don't worry about the format. I'll just reformat once it is back in master.

What about the test failures though?

**Comment by kipz** [comment]
tests schmests

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [1.0.0-m.2] - 2017-05-09
 
+### Added
```
Comment (cdupuis) [inline CHANGELOG.md]: That appears to be in the wrong spot, isn't it?

File: src/main/java/com/atomist/rug/cli/command/describe/DescribeCommand.java [hunk 2]
```diff
@@ -336,7 +342,7 @@ private void describeInvokeArchive(ArtifactDescriptor artifact) {
         log.newline();
         log.info("To get more information on any of the Rugs listed above, run:");
         log.info(
-                "  %s%s editor|generator|command-handler|event-handler|response-handler|function %s",
+                "  %s%s editor|generator|command-handler|event-handler|response-handler|integration-test|function %s",
```
Comment (cdupuis) [inline src/main/java/com/atomist/rug/cli/command/describe/DescribeCommand.java]: I think there is a similar addition need in `DescribeCommandInfo`.

**Pull request merged**